content/docs/getting-started.md

```diff
@@ -18,109 +18,110 @@ redirect_from:
 
-## Staying Informed {#staying-informed}
+## Haberdar olun {#staying-informed}
 
-The [React blog](/blog/) is the official source for the updates from the React team. Anything important, including release notes or deprecation notices, will be posted there first.
+[React blog](/blog/) React ekibinden yapılan güncellemelerin resmi kaynağıdır. Sürüm notları veya itiraz bildirimleri dahil olmak üzere, önemli olan her şey önce buraya gönderilecektir.
 
-You can also follow the [@reactjs account](https://twitter.com/reactjs) on Twitter, but you won't miss anything essential if you only read the blog.
+Twitter'da [@reactjs hesabını](https://twitter.com/reactjs) da takip edebilirsiniz, ama sadece blogu okursanız önemli hiç bir şeyi kaçırmazsınız.
 
-Not every React release deserves its own blog post, but you can find a detailed changelog for every release [in the `CHANGELOG.md` file in the React repository](https://github.com/facebook/react/blob/master/CHANGELOG.md), as well as on the [Releases](https://github.com/facebook/react) page.
+Her React sürümü kendi blog gönderisini hak etmiyor ama her sürüm için yapılan değişikliklerin ayrıntıarını [React havuzunun içindeki `CHANGELOG.md` dosyasında](https://github.com/facebook/react/blob/master/CHANGELOG.md) ve [Sürümler](https://github.com/facebook/react) sayfasında bulabilirsiniz.
 
-## Versioned Documentation {#versioned-documentation}
+## Versiyon Anlatımı {#versioned-documentation}
 
-This documentation always reflects the latest stable version of React. Since React 16, you can find older versions of the documentation [on a separate page](/versions). Note that documentation for past versions is snapshotted at the time of the release, and isn't being continuously updated.
+Bu bölüm her zaman React'ın en son kararlı versiyonunu yansıtır. React 16'dan beri, açıklamaların eski sürümlerini [ayrı bir sayfada](/versions) bulabilirsiniz. Geçmiş sürümlere ait açıklamaların yayınlanma tarihinde anlık olarak görüntülendiğini ve sürekli güncellenmediğini unutmayın.
 
-## Something Missing? {#something-missing}
+## Bir şeyler mi eksik? {#something-missing}
 
-If something is missing in the documentation or if you found some part confusing, please [file an issue for the documentation repository](https://github.com/reactjs/reactjs.org/issues/new) with your suggestions for improvement, or tweet at the [@reactjs account](https://twitter.com/reactjs). We love hearing from you!
+Anlatımlarda bir şey eksikse veya kafa karıştırıcı bir bölüm bulursanız, lütfen düzeltme/geliştirme önerilerinizle [ilgili dokümantasyon havuzuna bir sorun](https://github.com/reactjs/reactjs.org/issues/new) gönderin veya [@reactjs hesabına](https://twitter.com/reactjs) tweet atın. Sizden bildirim almayı çok seviyoruz!
```

Comment:
ilgili dokümantasyon havuzuna bir sorun
yerine
ilgili issue 
daha uygundur.

Reply:
I couldn't understand. Do you mean >> 'geliştirme önerilerinizle [ilgili issue] gönderin' instead of 'geliştirme önerilerinizle [ilgili dokümantasyon havuzuna bir sorun] gönderin'?

Reply:
Yep.  'geliştirme önerilerinizle [ilgili issue] gönderin'  is correct.